### PR TITLE
more tests in LocalBackendSpec, API change to CromwellTestkitSpec

### PR DESCRIPTION
**Pull request by kcibul**

### CODE REVIEW

**Comment by geoffjentry** [comment]
👍

**Comment by mcovarr** [comment]
:+1:

File: engine/src/test/scala/cromwell/engine/backend/local/LocalBackendSpec.scala [hunk 1]
```diff
@@ -33,45 +23,43 @@ class LocalBackendSpec extends CromwellTestkitSpec with Mockito with WorkflowDes
...
     }
 
-    "allow stderr if failOnStderr is not set" ignore {
-      testFailOnStderr(stderrDescriptor("runtime {failOnStderr: false}"), expectSuccess = true)
+    "allow stderr if failOnStderr is not set" in {
+      runWdl(
+        sampleWdl = StderrWdl,
+        eventFilter = EventFilter.info(pattern = "Workflow complete", occurrences = 1),
+        runtime = "runtime {failOnStderr: false}",
+        terminalState = WorkflowSucceeded )
```
Comment (kshakir) [inline engine/src/test/scala/cromwell/engine/backend/local/LocalBackendSpec.scala]: There is an extra space before the closing `)` in the multiple `WorkflowSucceeded )` only. The above `WorkflowFailed)` is fine.

File: engine/src/test/scala/cromwell/engine/backend/local/LocalBackendSpec.scala [hunk 2]
```diff
@@ -33,45 +23,43 @@ class LocalBackendSpec extends CromwellTestkitSpec with Mockito with WorkflowDes
       """.stripMargin.replaceAll("RUNTIME", runtime)
     override val rawInputs =  Map.empty[String, String]
   }
+}
+class LocalBackendSpec extends CromwellTestkitSpec {
```
Comment (kshakir) [inline engine/src/test/scala/cromwell/engine/backend/local/LocalBackendSpec.scala]: Newline between the `object` and `class`.

**Comment by kshakir** [comment]
Very minor fixes, then go for single 👍  merge.

**Pull request merged**